dashboard/lib/pd/jot_form/datetime_question.rb

```diff
@@ -0,0 +1,19 @@
+module Pd
+  module JotForm
+    class DatetimeQuestion < Question
+      def self.supported_types
+        [
+          TYPE_DATETIME
+        ]
+      end
+
+      def answer_type
+        ANSWER_DATETIME
+      end
+
+      def get_value(answer)
+        answer
```

Comment:
Should we merge individual date,month,year into 1 datetime string?

Reply:
Fair point.  I've made this change now.